Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs

```diff
@@ -367,12 +367,18 @@ public void OnInputDown(InputEventData eventData)
                 }
             }
 #else
-            if (!eventData.InputSource.SupportsInputInfo(eventData.SourceId, SupportedInputInfo.Position))
+            if (!eventData.InputSource.SupportsInputInfo(
+                eventData.SourceId, 
+#if UNITY_2017_1_OR_NEWER
+                SupportedInputInfo.PointerPosition))
+#else
+                SupportedInputInfo.Position))
+#endif // UNITY_2017_1_OR_NEWER
             {
                 // The input source must provide positional data for this script to be usable
                 return;
             }
-#endif
+#endif // UNITY_2017_2_OR_NEWER
```

Comment:
is this right?

Reply:
nope. i fixed it in https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2061/commits/fb3fc5086606be1a5e3c9aacc3d01e6cc843187e

Reply:
This matches line 358: https://github.com/davidkline-ms/MixedRealityToolkit-Unity/blob/fb3fc5086606be1a5e3c9aacc3d01e6cc843187e/Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs#L358

Reply:
![image](https://user-images.githubusercontent.com/13334553/39786528-d88c3c9c-52d5-11e8-8f6d-ef9bc0c8cac1.png)

Reply:
![image](https://user-images.githubusercontent.com/3580640/39787130-b3885bee-52d8-11e8-8922-1623d6b86e6a.png)

Reply:
Also, your red lines don't match up. There's an `#endif` between them

Reply:
Looks good for me now.